jest.config.js

```diff
@@ -1,7 +1,6 @@
 /* eslint-env node */
 module.exports = {
   testEnvironment: 'jsdom',
-  moduleFileExtensions: ['ts', 'js', 'tsx'],
```

Comment:
The default already covers relevant files: 

`["js", "mjs", "cjs", "jsx", "ts", "tsx", "json", "node"]`

see https://jestjs.io/docs/next/configuration#modulefileextensions-arraystring